terraso_backend/apps/project_management/graphql/site_notes.py

```diff
@@ -40,6 +42,8 @@ class Meta:
 class SiteNoteAddMutation(BaseWriteMutation):
     site_note = graphene.Field(SiteNoteNode, required=True)
 
+    model_class = SiteNote
```

Comment:
The site note mutations were previously missing a model class declaration, which caused issues when testing failure cases. I still suspect that they aren't fully implemented correctly, as I wasn't able to get them to return error codes in those cases, but that seems like a task out-of-scope for this ticket.